Bypass mlx5 extended statistics.
Subject: mlx5: Bypass extended stats.

Bypass mlx5 extended stats temporarily as this routine causes the mlx5 driver
to crash when loading the driver.

Signed-off-by: Sameer Agrawal <sagrawa@vyatta.att-mail.com>
--- a/drivers/net/mlx5/mlx5_stats.c
+++ b/drivers/net/mlx5/mlx5_stats.c
@@ -312,6 +312,7 @@ int
 mlx5_xstats_get(struct rte_eth_dev *dev, struct rte_eth_xstat *stats,
 		unsigned int n)
 {
+	return 0;
 	struct mlx5_priv *priv = dev->data->dev_private;
 	unsigned int i;
 	uint64_t counters[n];
